external.bzl

```diff
@@ -38,10 +38,10 @@ def _rule_dependencies():
     # Specifically define and register only Java toolchains we intend to support.
     remote_jdk17_repos()
     remote_jdk19_repos()
-    remote_jdk20_repos()
+    #remote_jdk20_repos()
```

Comment:
AFAIK, Kythe still needs to support Java 20.

Reply:
Noted. I think this is because nix for some reason does not seem to have jdk20... not sure why, and maybe I just didn't quite get to the bottom of this.

Reply:
Reverted.